src/squidpy/datasets/_10x_datasets.py

```diff
@@ -134,3 +142,54 @@ def visium(
         )
 
     return read_visium(base_dir / sample_id)
+
+
+def visium_hne_sdata(folderpath: Path | str) -> sd.SpatialData:
```

Comment:
Good PR! The only comment I have is here: this function is not really Visium specific and replicating this for other techs would lead to code redundancy. I would turn this function into a more general one and pass the Visium-specific strings as argument.

Reply:
Refactored it 👍